src/Sylius/Bundle/AdminBundle/Resources/config/services/controller.xml

```diff
@@ -12,6 +12,10 @@
 -->
 
 <container xmlns="http://symfony.com/schema/dic/services" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+    <parameters>
+        <parameter key="sylius.admin.notification.uri">http://www.sylius-hub.org/sylius/version</parameter>
```

Comment:
Are we sure about using sylius.**com**? Currently it redirects to sylius.**org** :)